core/server/proxy/src/main/java/alluxio/proxy/s3/S3RestUtils.java

```diff
@@ -98,8 +98,10 @@ public static <T> Response call(String resource, S3RestUtils.RestCallable<T> cal
           case NO_CONTENT:
             return Response.noContent().build();
           default:
-            return S3ErrorResponse.createErrorResponse(new S3Exception(
-                "Response status is invalid", resource, S3ErrorCode.INTERNAL_ERROR), resource);
+            return S3ErrorResponse.createErrorResponse(new S3Exception(resource,
+                new S3ErrorCode(S3ErrorCode.INTERNAL_ERROR.getCode(),
+                    "Response status is invalid",
+                    S3ErrorCode.INTERNAL_ERROR.getStatus())), resource);
```

Comment:
No semantic change, just syntax due to the method signature not existing in `branch-2.8` yet.